recipes/hwloc/all/conanfile.py

```diff
@@ -0,0 +1,114 @@
+from conan import ConanFile
+from conan.tools.apple import is_apple_os, fix_apple_shared_install_name
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rm, rmdir
+from conan.tools.gnu import Autotools, AutotoolsToolchain, PkgConfigDeps
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class HwlocConan(ConanFile):
+    name = "hwloc"
+    description = "Portable Hardware Locality (hwloc)"
+    topics = ("hwloc")
```

Comment:
Topics that just repeat the library name are redundant (It's used for the search terms in conan center index), ones that use keywords that describe the library are generally much better :)

Reply:
do we have a guarantee that after fixing this comment the recipe will be merged?

I have already spent a week with fixing _minor_ comments from different reviewers (note, that recipe is green for a while). It's very annoying .. why cannot maintainers write all comments at once? Why every new person comes to the PR every two days and imagines more and more minor comments? Is it never ending loop?

Reply:
removed `topic` at all

Reply:
great.. this useless `topics` attribute is mandatory
Really? Who actually uses this tags? Everybody searches in google, not via Conan topics.

Reply:
> why cannot maintainers write all comments at once? Why every new person comes to the PR every two days and imagines more and more minor comments? Is it never ending loop?

Let's discuss this better in your issue that deals about how you feel about the review process, I think it's a better place :)

> do we have a guarantee that after fixing this comment the recipe will be merged?

But no. If the remaining reviewer(s) find something else I might have missed, the review will continue, it's the whole point of the reviews! Again, let's better discuss this in the linked issue

> great.. this useless topics attribute is mandatory

I understand your frustration about what feel like minor stylistic choices that have nothing to do with the really important parts. We're actively working on improving contributor flows for such things and we appreciate the input, but yeah as I have mentioned, topics *are* used as a search term in Conan Center, and it's still a feature we want to support, that's why we ask for them to be useful :)

Reply:
> topics are used as a search term in Conan Center, and it's still a feature we want to support, that's why we ask for them to be useful in most cases :)

If you want this to be a Conan feature, anyway you will have to review all other recipes on topic attribute, because right now other recipes has the same duplication. 

> But no. If the remaining reviewer(s) find something else I might have missed, the review will continue

How many reviewers should review? Could _you_ please approve the PR? You are the 4th one who wrote comments, but nobody from the previous ones either finally approved the PR or wrote new comments.

Reply:
> because right now other recipes has the same duplication.

See the other issue where I touch upon this. Topics have changed over time, and old recipes that get not too many updates end up desynced with our current thinkings, and that's not a bad thing. Don't focus on old recipes having outdated criteria, it would hamper our ability to evolve in the end

> How many reviewers should review? 

> How is a final decision [made]?

See also the linked issue where I explain this a bit better, but you're right in that it should be made more clear what the expectations for reviews are, whose voice matters in the end, and how the process flows

> is it a sane strategy?

Please understand that behind every decision and review there are real people and we are sometimes fallible, and we might disagree on what the best way forward is


Having said that, I think we have discussed topics a lot. We'll internally review your comments about the process and will let you know once we act on those, but for now please add _some_ topics so that we can approve the PR and get it finally merged, thanks! :)

Reply:
> some topics so that we can approve the PR and get it finally merged, thanks! :)

Already, hope this helps to get your 👍

Reply:
Thanks a lot!